src/lib.rs

```diff
@@ -275,6 +490,41 @@ mod tests {
 
     use super::*;
 
+    #[tokio::test]
+    async fn verify_vc() {
+        let json_vc = include_str!("../tests/res/vc");
+        let result = verify_json_vc_string(json_vc.into()).await.is_ok();
```

Comment:
In general you don't assert that a `Result` is `Ok` -- you simply `unwrap()` it which has to benefit of printing the error for debugging, but it's not a necessary change if it works